pom.client.build.xml

```diff
@@ -35,6 +35,6 @@
 
   <modules>
     <module>./template/client</module>
-    <module>./keyvault/data-plane</module>
+    <module>./batch/data-plane</module>
```

Comment:
This is probably not what you want to do :-)

Reply:
It is fixed in pom.client.xml
This file is removed.